microsoft/MixedReality-WorldLockingTools-Unity
# WLT AlignSubTree SpacePin Mesh Visualization feature

Steps to test the feature:

1. Drag the SpacePinVisualizer prefab into the root of the SpacePinPartial scene.
2. Create SpacePins by manipulating the spheres.
3. Debug visuals should appear showing the weights and distribution of the currently active pins.
